module build fails for Linux 6.10

```
/var/lib/dkms/dm-writeboost/2.2.17/build/dm-writeboost-target.c: In function 'dm_devsize':
/var/lib/dkms/dm-writeboost/2.2.17/build/dm-writeboost-target.c:119:37: error: 'struct block_device' has no member named 'bd_inode'
  119 |         return i_size_read(dev->bdev->bd_inode) >> 9;
      |                                     ^~
```

The corresponding Linux commit is https://github.com/torvalds/linux/commit/203c1ce0bb063d1620698e39637b64f2d09c1368 "RIP ->bd_inode" in v6.10-rc1.

--- a/src/dm-writeboost-target.c
+++ b/src/dm-writeboost-target.c
@@ -116,7 +116,11 @@ int wb_io_internal(struct wb_device *wb, struct dm_io_request *io_req,
 
 sector_t dm_devsize(struct dm_dev *dev)
 {
+#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 10, 0)
 	return i_size_read(dev->bdev->bd_inode) >> 9;
+#else
+	return bdev_nr_sectors(dev->bdev);
+#endif
 }
 
 /*----------------------------------------------------------------------------*/

--- a/src/dm-writeboost-target.c
+++ b/src/dm-writeboost-target.c
@@ -116,7 +116,7 @@ int wb_io_internal(struct wb_device *wb, struct dm_io_request *io_req,
 
 sector_t dm_devsize(struct dm_dev *dev)
 {
-#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 10, 0)
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 11, 0)
 	return i_size_read(dev->bdev->bd_inode) >> 9;
 #else
 	return bdev_nr_sectors(dev->bdev);

